src/Juvix/Backends/Michelson/DSL/InstructionsEff.hs

```diff
@@ -26,6 +26,7 @@ import qualified Juvix.Core.Usage as Usage
 import Juvix.Library hiding (abs, and, or, xor)
 import qualified Juvix.Library (abs)
 import qualified Michelson.Untyped.Instr as Instr
+import qualified Michelson.Untyped.Type as MU
```

Comment:
We should be able to do this without importing this. 

```haskell
Juvix.Backends.Michelson.DSL.Untyped
```
should be used instead

Reply:
Right, fixed.